clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -349,30 +349,27 @@ public void handle(JoinGroupResponse joinResponse, RequestFuture<ByteBuffer> fut
             } else if (errorCode == Errors.UNKNOWN_MEMBER_ID.code()) {
                 // reset the member id and retry immediately
                 AbstractCoordinator.this.memberId = JoinGroupRequest.UNKNOWN_MEMBER_ID;
-                log.info("Attempt to join group {} failed due to unknown member id, resetting and retrying.",
-                        groupId);
+                log.debug("Attempt to join group {} failed due to unknown member id, resetting and retrying.", groupId);
```

Comment:
It happens elsewhere. We could probably just leave that part off if we're reducing the level to debug. Seems like it was originally intended to give users some comfort that the error is being handled internally.